components/perl/Clone/perl-integrate-module.conf

```diff
@@ -0,0 +1,16 @@
+%hook-no-license%
```

Comment:
Why do you need `%hook-no-license%`?  AFAIK, it is never executed for Clone module.

And, this file still lacks the license header.  By license header I mean this:

```
$ head -n15 components/perl/Authen-PAM/perl-integrate-module.conf
#
# This file and its contents are supplied under the terms of the
# Common Development and Distribution License ("CDDL"), version 1.0.
# You may only use this file in accordance with the terms of version
# 1.0 of the CDDL.
#
# A full copy of the text of the CDDL should have accompanied this
# source.  A copy of the CDDL is also available via the Internet at
# http://www.illumos.org/license/CDDL.
#

#
# Copyright 2022 Marcel Telka
#

$
````

Reply:
because of this:

`head Clone.license
---------

Copyright 2001-2019 Ray Finch. All Rights Reserved.

This module is free software; you can redistribute it and/or
modify it under the same terms as Perl itself.

===========================================================================

This program is free software; you can redistribute it and/or modify
`
If it makes you happy I added the header.